test/Parse/ConditionalCompilation/can_import_submodule_missing_requirement.swift

```diff
@@ -0,0 +1,22 @@
+// RUN: %target-typecheck-verify-swift -I %S/Inputs/can-import-submodule-missing-requirement/ 
+
+#if canImport(A.BMissingRequirement)
+#error("should not can import A.BMissingRequirement")
+#endif
```

Comment:
I'm thinking about how to make sure `canImport` failed because of missing requirements, not because of miss-spelling, etc. Therefore, I want to write test like:

```swift
#if canImport(A.BMissingRequirement)
#error("should not can import A.BMissingRequirement") // expected-error {{no such module 'A.BMissingRequirement'}}
#else
import A.BMissingRequirement
#endif
```

However, I'm not quite sure how to check for the error `module 'A.BMissingRequirement' requires feature 'missing'` diagnosed at `%S/Inputs/can-import-submodule-missing-requirement/module.modulemap:7:21`.

Reply:
I wouldn't worry too much about it. External diagnostics are a known shortfall of the verifier's model.

Reply:
Alright then